backend/prompt_studio/prompt_studio_core_v2/exceptions.py

```diff
@@ -67,3 +67,22 @@ class MaxProfilesReachedError(APIException):
         f"Maximum number of profiles (max {ProfileManagerKeys.MAX_PROFILE_COUNT})"
         " per prompt studio project has been reached."
     )
+
+
+class OperationNotSupported(APIException):
+    status_code = 403
```

Comment:
@chandrasekharan-zipstack Handled this as a part of the app. https://github.com/Zipstack/unstract/pull/720

Reply:
We'll take care of it if there are any conflicts